packages/cli/generators/model/templates/model.ts.ejs

```diff
@@ -5,7 +5,7 @@ import {model, property} from '@loopback/repository';
 import {<%= modelBaseClass %>} from '.';
 <% } -%>
 
-@model()
+@model(<%- locals.settings ? JSON.stringify({settings}) : '' %>)
```

Comment:
Where is `locals.settings` coming from? Did you perhaps mean to check `settings`?

Reply:
Yes, this was put in to check if the settings object existed.  Is there a more idiomatic way to do this?

Reply:
Have you tried to following?

```suggestion
@model(<%- settings ? JSON.stringify({settings}) : '' %>)
```

Reply:
It will throw a reference error if it's not there, I got my solution from here

https://stackoverflow.com/questions/5372559/what-is-the-proper-way-to-check-for-existence-of-variable-in-an-ejs-template-us

Reply:
`locals` is special variable for EJS and it's safer to use `locals.settings` if `settings` can be `undefined` when the EJS template is rendered.

Reply:
Makes sense, I was not aware of this aspect of EJS.

Thank you for being patient with me and explaining the details, I learned something new today 😄